src/Operator/Base.jl

```diff
@@ -92,6 +92,7 @@ mutable struct Operator
     multiplicity::AbstractArray{Float64}
     rowmodemap::AbstractArray{Float64,2}
     columnmodemap::AbstractArray{Float64,2}
+    qtbtd::AbstractArray{Float64,2}
```

Comment:
Can we pick a less cryptic name? Is this for half assembly?

Reply:
Ja genau, ```qtbtd``` seems too cryptic